.vscode/settings.json

```diff
@@ -1,20 +1,7 @@
 {
     "rust-analyzer.linkedProjects": [
-        // Root Workspace & CLIs.
         "Cargo.toml",
         "radix-clis/Cargo.toml",
-        // Various blueprints we have.
```

Comment:
These slowed down the IDE performance greatly for me. Every save took minutes to run and each check took a lot longer.

Reply:
Ooooo is that why my rust analyzer has been stuck on a crash loop since merging in develop yesterday? :D

I've tried `cargo clean`, `rustup update`, changing my rust analyzer version... I'm glad removing this fixes it.

Reply:
Yup, I sadly think that this is what's causing it :/ Bit of a shame to be honest since it's nice to get diagnostics in your IDE for these blueprints especially when performing large refactors. Only alternative is the the `cargo_check.py` script that I added but it doesn't give us any diagnostics in the IDE, just command line.